README.md

```diff
@@ -1,2 +1,16 @@
 # Crossword Master
-Crossword Master is a web crossword generator &amp; solver with an intuitive UI.
+[Crossword Master](https://crossword-master.org) is a web crossword generator &amp; solver with an intuitive UI.
+
+# What it can do
+
+https://user-images.githubusercontent.com/46112160/209452638-acf2a2db-278c-4ada-a312-7c8cdbf92daa.mp4
+
+# Technologies used
+
+This project uses `typescript` on frontend, `python` on backend and:
```

Comment:
“The app is written in TypeScript/Python and makes use of the following technologies:”